test/chtest-make-auto

```diff
@@ -40,8 +56,7 @@ tag_from_path () {
 template_print () {
     template="./make-auto.d/$1"
 
-    cat "$template" | sed "s@%(addr)s@$addr@g" \
-                    | sed "s@%(basename)s@$basename_@g" \
+    cat "$template" | sed "s@%(basename)s@$basename_@g" \
                     | sed "s@%(dirname)s@$dirname_@g" \
                     | sed "s@%(path)s@$path_@g" \
                     | sed "s@%(scope)s@$scope@g" \
```

Comment:
I'd say we should make an issue to change these and related pipes to a single sed/awk script.